app/models/permission.rb

```diff
@@ -96,8 +96,17 @@ def self.project_list(project_id)
   def self.filter_settings
     {
       valid_fields: [:id, :project_id, :user_id, :level, :allow_anonymous, :allow_logged_in, :creator_id, :created_at],
-      render_fields: [:id, :project_id, :user_id, :level, :allow_anonymous, :allow_logged_in],
+      render_fields: [:id, :project_id, :user_id, :level, :allow_anonymous, :allow_logged_in, :updated_at, :updater_id, :created_at, :creator_id],
```

Comment:
A PR for another day, but these new params should probably be an array somewhere and merged in here instead of redeclared throughout baw-server